### DISCUSSION TITLE
Type Error while running transactWrite function on dynamodb

### DISCUSSION THREAD

**Discussion by deeksha-schneider** [aws/aws-sdk-js#4465] [Q&A]
"TypeError: require(...).v4 is not a function\n    
at Object.uuidV4 [as v4] (\node_modules\\aws-sdk\\lib\\util.js:951:30)\n    
at Request.BUILD_IDEMPOTENCY_TOKENS (\\node_modules\\aws-sdk\\lib\\event_listeners.js:154:56)\n  
  at Request.callListeners
   (\node_modules\\aws-sdk\\lib\\sequential_executor.js:106:20)\n   
    at callNextListener node_modules\\aws-sdk\\lib\\sequential_executor.js:96:12)\n    at \\

**Comment by ajredniwja** [comment]
Hi @deeksha-schneider, its hard to say what might be causing this issue, it seems like a coding error, can you please share the code you are using?
